modules/catalog/src/main/java/org/apache/ignite/internal/catalog/storage/NewColumnsEntry.java

```diff
@@ -49,18 +49,19 @@ public class NewColumnsEntry implements UpdateEntry, Fireable {
 
     private final int tableId;
     private final List<CatalogTableColumnDescriptor> descriptors;
-    private final String schemaName;
+    private final int schemaId;
```

Comment:
I would delete schema at all because it creates ambiguity. It's better to derive schema from table (table descriptor has `schemaId`). 

The same for rest of the updateEntries

Reply:
Fixed.